packages/@okta/vuepress-site/docs/concepts/multi-tenancy/index.md

```diff
@@ -117,11 +117,11 @@ Host tenants in a single org using Universal Directory (UD)
 
 ### Summary
 
-This configuration hosts all tenants in a single org. Customers have two
-ways to do this:
+This configuration hosts all tenants in a single org. Customers have one
+way to do this:
```

Comment:
Editorially, I'd suggest we don't have to introduce the only way to do something with a setup line like "Customers have one way to do this:"

But looking at the content below, it looks like it's intentionally commented our instead of removed? Do I understand that right or can we just rephrase this opener?

Reply:
Yes, I totally agree. And you are correct. I commented out the text and kept the structure in place only because it will be readded when the DAC replacement becomes available. If you want it rephrased i can do that too. We'll just need to place it back into bullets when the other option get's readded again.

Reply:
@grahamsmith-okta I've updated the text in the latest text based on your feedback.

Reply:
Looks great, thanks. I think this reads a little cleaner now